src/HealthChecks.UI/Core/HostedService/HealthCheckReportCollector.cs

```diff
@@ -94,9 +93,13 @@ private async Task<UIHealthReport> GetHealthReportAsync(HealthCheckConfiguration
                 var absoluteUri = GetEndpointUri(configuration);
 
                 using var response = await _httpClient.GetAsync(absoluteUri, HttpCompletionOption.ResponseHeadersRead);
-                //response.EnsureSuccessStatusCode(); TODO: add or not ?

```

Comment:
Are you sure? What are your considerations?

Reply:
Unhealthy status returns HTTP status code 503, when we use EnsureSuccessStatusCode() it will throws exception and we wouldn't be able to process the response correctly.

Reply:
You're right, I forgot about it.